### PR TITLE
Hotfix: add firewall rules to sync monitor and SSL reload support 

### PR DESCRIPTION
**Pull request by dmytrotkk**

### CODE REVIEW

File: web/models/schain.py [hunk 1]
```diff
@@ -164,6 +167,23 @@ def reset_failed_conunters(self) -> None:
         self.set_restart_count(0)
         self.set_failed_rpc_count(0)
 
+    def set_ssl_change_date(self, value: datetime) -> None:
+        logger.info(f'Changing ssl_change_date for {self.name} to {value}')
+        self.ssl_change_date = value
+        self.save()
+
+    def ssl_reloaded(self) -> None:
+        ssl_files_change_date = get_ssl_files_change_date()
```
Comment (badrogger) [inline web/models/schain.py]: I think It's better to pass change data as a parameter. There is no need to keep additional dependency here

File: web/models/schain.py [hunk 2]
```diff
@@ -164,6 +167,23 @@ def reset_failed_conunters(self) -> None:
...
+        self.set_ssl_change_date(ssl_files_change_date)
+
+    def ssl_reload_needed(self) -> bool:
```
Comment (badrogger) [inline web/models/schain.py]: It should not be in the scope of model.

File: core/schains/ssl.py [hunk 3]
```diff
@@ -31,3 +32,10 @@ def get_ssl_filepath():
     else:
         return os.path.join(SSL_CERTIFICATES_FILEPATH, 'ssl_key'), \
             os.path.join(SSL_CERTIFICATES_FILEPATH, 'ssl_cert')
+
+
+def get_ssl_files_change_date() -> datetime:
+    if is_ssl_folder_empty():
+        return
+    ssl_changed_ts = os.path.getmtime(SSL_CERT_PATH)
+    return datetime.fromtimestamp(ssl_changed_ts)
```
Comment (badrogger) [inline core/schains/ssl.py]: datetime.utcfromtimestamp is better

File: core/schains/monitor/sync_node_reload_monitor.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+#   along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import logging
+
+from core.schains.monitor import BaseMonitor
+
+logger = logging.getLogger(__name__)
+
+
+class SyncNodeReloadMonitor(BaseMonitor):
```
Comment (badrogger) [inline core/schains/monitor/sync_node_reload_monitor.py]: It's absolutely the same as regular reload monitor except sync_node=True for `self.reloaded_skaled_container(sync_node=True)`. Probably it make sense to set sync_node as class attribute.

**Review by badrogger — CHANGES_REQUESTED** [review]

File: web/models/schain.py (hunk 1 above)
Comment (dmytrotkk) [reply web/models/schain.py]: for this we have `set_ssl_change_date`, I'll move `ssl_reloaded` from class to helper methods

File: core/schains/ssl.py (hunk 3 above)
Comment (dmytrotkk) [reply core/schains/ssl.py]: ++

File: core/schains/monitor/sync_node_reload_monitor.py (hunk 4 above)
Comment (dmytrotkk) [reply core/schains/monitor/sync_node_reload_monitor.py]: not exactly.
for all regular node monitors, we have `@BaseMonitor.monitor_runner` decorator for `run` method.
for sync node we don't.

File: web/models/schain.py (hunk 2 above)
Comment (dmytrotkk) [reply web/models/schain.py]: ++

**Review by badrogger — APPROVED** [review]

**Pull request merged**